active/0009-import-export-feature/README.md

```diff
@@ -0,0 +1,340 @@
+- Start Date: 2024-11-12
+- RFC PR: https://github.com/datahub-project/rfcs/pull/9
+- Implementation PR(s): 
+
+# Import/Export Feature
+
+## Summary
+
+This feature will add the ability to both export datasets to CSV files and import them back into DataHub from those CSV files, using the UI. Code is already implemented for this feature, though further work may need to be done. This RFC details the Implementation in its current state.
+
+## Motivation
```

Comment:
A section on the User Journey and Motivations would be useful here. 

e.g. 
Why is the User exporting to csv -> Do they intend to make some changes to metadata in bulk and then import it back? 

Are there scenarios where Users are trying to import CSV-s containing metadata that has been hand written or sourced from "non DataHub" catalogs? In those scenarios - how will users provide the "urn" field which represents the main identity of the dataset on DataHub?